src/io/github/perrymant/moneymaker/Moneymaker.java

```diff
@@ -1,10 +1,25 @@
 package io.github.perrymant.moneymaker;
 
+import java.io.IOException;
+
 public class Moneymaker {
 
     private static final Budget budget = new Budget(new Balance(), new TransactionMaker().getTransactions());
 
-    public static void main(String[] args) {
-        new Logger().info(budget.report());
+    public static void main(String[] args) throws IOException {
```

Comment:
`main` should never throw. Handled exceptions (the ones the IDE says you have to throw) are evil and should be wrapped in a `RuntimeException` as low down as possible instead.

As for solving the length of main here:
- Create a new class called Main and put the main method here. This makes it easy to spot when looking for it in the project view.
- Create another class `Application` or `MoneyMaker` where you put this logic inside a `start(String[] args)` method.
- Change main to be
```java
public static void main(String[] args) {
    new Application().start();
}
```